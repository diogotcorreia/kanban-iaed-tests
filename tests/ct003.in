t 1 zqz  ozauuv   ejgulb   xzeq mnfqxucshcr  h feqcduj
t 2 fhngwpneuaybnpafrlnkmzonuoqxrwrhbcfhdbtbdxoaaxsgwq
t 3 mafkbmoaifchtidyywdqhenurgn      hwjjydgwlpwfdyuiu
t 4 nkzlf   iyav  xbg  crxpndixbuujtmpdevzncubmqdrsgma
t 5 parbqvsrbjfhxosghltcmltfsvnllqxedqlelrghkboyzfnumk
t 6 a       zjzbvcugixcwvkpyyefmzakxvmmaqtfbelh      y
t 7 zcpikszlmdjaejxyathofguudozcuiaabfhgokzqlfxjcpgmft
t 8 zmvrqvqvgnphewotahtlmomunrltggowwiyrmifabchfvneuqf
t 9 wfbgbubxxzncwmnhwkueonfbvsdcgfvzalhdivyrsjuiihvvxg
t 10 ezcsnyisfdtyrfesdiuqfbcjgcckefhkcwrfduvdauacbuoelo
t 11 ta  fmjtfy   lcewjr   doy  sdvitzklzrjpsdemxoojmzl
t 12 jq  gunpqrudyaw  bmlnaawdingny  iylzhm nngygfarbek
t 13 nyewnqlosngaxnpxbfbrhxwvajtghbhkxaawhpyistvosnvsab
t 14 qldanqaoptgk                           xkovmwbstyd
t 15 k                    a                           v
u ozrijabjwfyqxztoadck
l
a JF         F       Y
a YXAOFGZMID BUJBIDOAC
a HOD      Z RNKA   GX
u adhkovjjgqbutagvplzv
a SGESEUAQZT TKBIHXIBB
a PDHDYUOFVV QXMXMBHUR
a XUHIUMHINK QSRNJAFFV
m 3 ozrijabjwfyqxztoadck JF         F       Y
a
q

