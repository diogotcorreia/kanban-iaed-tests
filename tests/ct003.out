task 1
task 2
task 3
task 4
task 5
task 6
task 7
task 8
task 9
task 10
task 11
task 12
task 13
task 14
task 15
6 TO DO #6 a       zjzbvcugixcwvkpyyefmzakxvmmaqtfbelh      y
10 TO DO #10 ezcsnyisfdtyrfesdiuqfbcjgcckefhkcwrfduvdauacbuoelo
2 TO DO #2 fhngwpneuaybnpafrlnkmzonuoqxrwrhbcfhdbtbdxoaaxsgwq
12 TO DO #12 jq  gunpqrudyaw  bmlnaawdingny  iylzhm nngygfarbek
15 TO DO #15 k                    a                           v
3 TO DO #3 mafkbmoaifchtidyywdqhenurgn      hwjjydgwlpwfdyuiu
4 TO DO #4 nkzlf   iyav  xbg  crxpndixbuujtmpdevzncubmqdrsgma
13 TO DO #13 nyewnqlosngaxnpxbfbrhxwvajtghbhkxaawhpyistvosnvsab
5 TO DO #5 parbqvsrbjfhxosghltcmltfsvnllqxedqlelrghkboyzfnumk
14 TO DO #14 qldanqaoptgk                           xkovmwbstyd
11 TO DO #11 ta  fmjtfy   lcewjr   doy  sdvitzklzrjpsdemxoojmzl
9 TO DO #9 wfbgbubxxzncwmnhwkueonfbvsdcgfvzalhdivyrsjuiihvvxg
7 TO DO #7 zcpikszlmdjaejxyathofguudozcuiaabfhgokzqlfxjcpgmft
8 TO DO #8 zmvrqvqvgnphewotahtlmomunrltggowwiyrmifabchfvneuqf
1 TO DO #1 zqz  ozauuv   ejgulb   xzeq mnfqxucshcr  h feqcduj
TO DO
IN PROGRESS
DONE
JF         F       Y
YXAOFGZMID BUJBIDOAC
HOD      Z RNKA   GX
SGESEUAQZT TKBIHXIBB
PDHDYUOFVV QXMXMBHUR
XUHIUMHINK QSRNJAFFV
